addons/l10n_fr_pdp_reports/models/pdp_flow.py

```diff
@@ -0,0 +1,752 @@
+import base64
+import logging
+import uuid
+from collections import defaultdict
+from lxml import etree
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_repr, html2plaintext
+from ..utils.vat import is_valid_vat
+
+
+_logger = logging.getLogger(__name__)
+
+
+class PdpFlow(models.Model):
+    """Represent a single Flux 10 payload that will be dispatched to the PDP proxy."""
+
+    _name = "l10n.fr.pdp.flow"
+    _description = "French PDP Flow"
+    _order = "create_date desc"
+    _sql_constraints = [
+        (
+            "unique_flow_batch",
+            "unique(company_id, reporting_date, currency_id, document_type, flow_type, transaction_type, batch_index, supplement_number)",
+            "A PDP e-reporting flow already exists for the same configuration.",
+        ),
+    ]
+
+    TRANSACTION_FLOW_TYPE = "transaction_report"
+    FLOW_SYNTAX_FRR = "FRR"
+    DEFAULT_FLOW_TYPE = TRANSACTION_FLOW_TYPE
+    OPEN_STATES = frozenset({"draft", "building", "ready", "error"})
+
+    TRANSPORT_STATUS_MAPPING = {
+        "ACCEPTED": "pending",
+        "DELIVERED": "done",
+        "ERROR": "error",
```

Comment:
TBH i don't really see the benefit of those, since you use them in some getter function, just use the getter function or directly the string if there is only one ?